src/web_app_skeleton/script/setup.ecr

```diff
@@ -30,14 +30,8 @@ if [ ! -f ".env" ]; then
   print_done
 fi
 
-notice "Creating the database"
-lucky db.create | indent
-
-notice "Verifying postgres connection"
-lucky db.verify_connection | indent
-
-notice "Migrating the database"
-lucky db.migrate | indent
+notice "Creating and Migrating the database"
+lucky db.setup | indent
```

Comment:
Does `lucky db.setup` raise a nice error like `db.verify_connection` does? I think it makes sense to make sure all the db tasks return a nice error if it can't connect. If we don't have a nice error for `db.setup` I think we should keep this as-is and switch to db.setup when the errors are nice.

But maybe they already are nice, in which case I think this LGTM :)

Reply:
It will try to create the DB first, and if it can't connect, it throws

```
Creating the database failed. It looks like Postgres is not running.
      Message from Postgres:
        #{message}
      Try this...
        ▸ Make sure Postgres is running
        ▸ Check your database configuration settings
```

I do like the error message `verify_connection` gives, but if the DB doesn't exist yet, then it fails, so you have to call `create_db` first anyway.

I could update the `verify_connection` so the DB name is optional. Then if you pass it, it would check if you can connect to that instance, and if it's not there, then it just checks that you can connect to that engine... (not sure how that works yet).